wireapp/wire-ios-mono
# feat: append system message when mls migration finalizes during ongoing call WPB-5348

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-5348" title="WPB-5348" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />WPB-5348</a>  [iOS] If there is an ongoing call when migration finalization occurs
  </td></table>
  <br />
 







----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

If migration to mls is finalized while a call is ongoing, inform the user by appending a system message.

### Testing

#### Test Coverage

- Unit test asserting wire call center appends the system message.

#### How to Test

1. Be in a proteus group conversation
2. Start migrating to mls. Make sure there is one client that doesn't support mls.
3. Be in a call
4. Update the last client to support mls, this will trigger the migration to complete.
5. Assert you see the new system message

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
